services/web/server/src/simcore_service_webserver/api/v0/openapi.yaml

```diff
@@ -134,6 +118,78 @@ paths:
                             type: object
                   error:
                     title: Error
+  /announcements:
+    get:
+      tags:
+        - announcements
+      summary: List Announcements
+      operationId: list_announcements
+      responses:
+        '200':
+          description: Successful Response
+          content:
+            application/json:
+              schema:
+                properties:
+                  data:
+                    items:
+                      properties:
+                        id:
+                          type: string
+                          title: Id
+                        products:
+                          items:
+                            type: string
+                            enum:
+                              - osparc
```

Comment:
Should this list really be hardcoded here? I guess it will change in the future

Reply:
yes, i do not like this either. products are dynamic. i took it "as-it-is" but probably will drop this and add a runtime validation when it applies

Reply:
ACTUALLY you have given me an idea. I will filter the announcments in the backend per product